### ISSUE TITLE
ADDON_ACTION_FORBIDDEN - TipTac tried to call the protected function 'UseAction()'

### ISSUE THREAD

**Issue by timetofall** [frozn/TipTac#141]
**Describe the bug**
Pop-up saying "TipTac has been blocked from an action only available to the Blizzard UI"
looks like UI taint, hearthed to Org after a dungeon and tried to press an ability on my action bar 

**To Reproduce**
looks like UI taint, hearthed to Org after a dungeon and tried to press an ability on my action bar 

**TipTac Reborn (please complete the following information):**
 - Version 22.11.20
 - WoW 10.0.2 Build 46702 (retail)

LUA Error:

> 1x [ADDON_ACTION_FORBIDDEN] AddOn 'TipTac' tried to call the protected function 'UseAction()'.
> [string "@!BugGrabber/BugGrabber.lua"]:480: in function <!BugGrabber/BugGrabber.lua:480>
> [string "=[C]"]: in function `UseAction'
> [string "@FrameXML/SecureTemplates.lua"]:364: in function `handler'
> [string "@FrameXML/SecureTemplates.lua"]:690: in function <FrameXML/SecureTemplates.lua:672>
> [string "@FrameXML/SecureTemplates.lua"]:704: in function <FrameXML/SecureTemplates.lua:697>
> [string "@FrameXML/SecureTemplates.lua"]:746: in function `SecureActionButton_OnClick'
> [string "@FrameXML/ActionButton.lua"]:109: in function `TryUseActionButton'
> [string "@FrameXML/MultiActionBars.lua"]:14: in function `MultiActionButtonDown'
> [string "MULTIACTIONBAR1BUTTON1"]:2: in function <[string "MULTIACTIONBAR1BUTTON1"]:1>
>

**Comment by tatatada** [comment]
1x [ADDON_ACTION_FORBIDDEN] AddOn 'TipTac' hat versucht die geschützte Funktion 'TargetUnit()' aufzurufen.
[string "@!BugGrabber/BugGrabber.lua"]:480: in function <!BugGrabber/BugGrabber.lua:480>
[string "=[C]"]: in function `TargetUnit'
[string "@FrameXML/EditModeManager.lua"]:1512: in function `RefreshTargetAndFocus'
[string "@FrameXML/EditModeManager.lua"]:1456: in function `OnEditModeEnter'
[string "@FrameXML/EditModeManager.lua"]:170: in function `EnterEditMode'
[string "@FrameXML/EditModeManager.lua"]:192: in function <FrameXML/EditModeManager.lua:190>
[string "=[C]"]: in function `Show'
[string "@FrameXML/UIParent.lua"]:2853: in function `SetUIPanel'
[string "@FrameXML/UIParent.lua"]:2657: in function `ShowUIPanel'
[string "@FrameXML/UIParent.lua"]:2561: in function <FrameXML/UIParent.lua:2557>
[string "=[C]"]: in function `SetAttribute'
[string "@FrameXML/UIParent.lua"]:3159: in function <FrameXML/UIParent.lua:3138>
[string "=[C]"]: ?
[string "=[C]"]: ?
[string "=[C]"]: in function `ShowUIPanel'
[string "*GameMenuFrame.xml:94_OnClick"]:2: in function <[string "*GameMenuFrame.xml:94_OnClick"]:1>


Happend for me too, i tried to edit the UI with the offical Dragonflight Edit Option and got this error firstplace

**Comment by jonjbm** [comment]
Just going to add mine as well for good measure:

```
2x [ADDON_ACTION_FORBIDDEN] AddOn 'TipTac' tried to call the protected function 'UseAction()'.
[string "@!BugGrabber/BugGrabber.lua"]:480: in function <!BugGrabber/BugGrabber.lua:480>
[string "=[C]"]: in function `UseAction'
[string "@FrameXML/SecureTemplates.lua"]:364: in function `handler'
[string "@FrameXML/SecureTemplates.lua"]:690: in function <FrameXML/SecureTemplates.lua:672>
[string "@FrameXML/SecureTemplates.lua"]:704: in function <FrameXML/SecureTemplates.lua:697>
[string "@FrameXML/SecureTemplates.lua"]:746: in function `SecureActionButton_OnClick'
[string "@FrameXML/ActionButton.lua"]:109: in function `TryUseActionButton'
[string "@FrameXML/MultiActionBars.lua"]:14: in function `MultiActionButtonDown'
[string "MULTIACTIONBAR3BUTTON2"]:2: in function <[string "MULTIACTIONBAR3BUTTON2"]:1>
```

**Comment by bargiddle** [comment]
`1x [ADDON_ACTION_FORBIDDEN] AddOn 'TipTac' tried to call the protected function 'UseAction()'.
[string "@!BugGrabber/BugGrabber.lua"]:480: in function <!BugGrabber/BugGrabber.lua:480>
[string "=[C]"]: in function `UseAction'
[string "@FrameXML/SecureTemplates.lua"]:364: in function `handler'
[string "@FrameXML/SecureTemplates.lua"]:690: in function <FrameXML/SecureTemplates.lua:672>
[string "@FrameXML/SecureTemplates.lua"]:704: in function <FrameXML/SecureTemplates.lua:697>
[string "@FrameXML/SecureTemplates.lua"]:746: in function `SecureActionButton_OnClick'
[string "@FrameXML/ActionButton.lua"]:109: in function `TryUseActionButton'
[string "@FrameXML/ActionButton.lua"]:144: in function `ActionButtonDown'
[string "ACTIONBUTTON2"]:2: in function <[string "ACTIONBUTTON2"]:1>`

for me at least it prevents almost all buttons from being pressed when it happens, so when it happens mid dungeon it kinda sucks.  other addons have done it too but tiptac seems to be the most often.  a /reload lets me use abilities again, but blegh

**Comment by frozn** [comment]
Should be fixed with latest release [v23.01.23](https://github.com/frozn/TipTac/releases/tag/v23.01.23-release). Please try again with this version.
